PurchasesTests/Networking/BackendTests.swift

```diff
@@ -770,14 +762,17 @@ class BackendTests: XCTestCase {
         })
 
         expect(eligibility).toEventuallyNot(beNil())
-        expect(eligibility!["producta"]!.status).toEventually(equal(IntroEligibilityStatus.unknown))
-        expect(eligibility!["productb"]!.status).toEventually(equal(IntroEligibilityStatus.unknown))
-        expect(eligibility!["productc"]!.status).toEventually(equal(IntroEligibilityStatus.unknown))
+
+        expect(eligibility?["producta"]?.status) == .unknown
```

Comment:
These were unnecessary, since the first one already waited for the response to finish.